src/styles/main.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calc((#{$day-size} * 7 + (6 * #{$gap})) + #{$padding} * 2);
+  padding: $padding;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $day-size;
+    width: $day-size;
+    background-color: #eee;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), color literals like `#eee` should be stored in variables for easier maintenance and to avoid repetition.